docs/guides/force-organizations.mdx

```diff
@@ -175,33 +179,52 @@ import { redirect, useParams } from "next/navigation"
 import { useAuth, useOrganizationList } from "@clerk/nextjs"
 
 export function SyncActiveOrganization() {
-  const { setActive, isLoaded } = useOrganizationList()
+  const { setActive, isLoaded } = useOrganizationList();
 
   // Get the organization ID from the session
-  const { orgId } = useAuth()
+  const { orgId } = useAuth();
 
   // Get the organization ID from the URL
-  const { orgId: urlOrgId } = useParams() as { orgId: string }
+  const { orgId: urlOrgId } = useParams() as { orgId: string };
 
   useEffect(() => {
-    if (!isLoaded) return
+    if (!isLoaded) return;
 
     // If the org ID in the URL is not valid, redirect to the homepage
     if (!urlOrgId?.startsWith("org_")) {
-      redirect("/")
-      return
+      redirect("/");
+      return;
     }
 
     // If the org ID in the URL is not the same as the org ID in the session (the active organization), set the active organization to be the org ID from the URL
     if (urlOrgId && urlOrgId !== orgId) {
-      void setActive({ organization: urlOrgId })
+      void setActive({ organization: urlOrgId });
```

Comment:
Is the `void` necessary there?

Reply:
it really depends on your eslint config, setActive returns a promise